vercel.json

```diff
@@ -1,12 +1,10 @@
 {
+  "name": "jsPerf",
+  "version": 2,
   "public": true,
-  "type": "docker",
-  "features": {
-    "cloud": "v2"
-  },
-  "alias": "jsperf.com",
+  "alias": ["jsperf.com"],
```

Comment:
`alias` and `name` are deprecated, the can be removed from `vercel.json`

Instead, you should add the production domain to your Vercel Project in the dashboard.